as4-client/src/main/java/dk/toldst/eutk/as4client/builder/interfaces/As4Optionals.java

```diff
@@ -12,5 +13,5 @@
     As4Optionals fromParty(String fromParty, String fromPartyRole);
     As4Optionals setAbsoluteURI(URI uri);
     As4Optionals setUsername(String username);
-    As4Client build() throws URISyntaxException;
+    As4Client build() throws URISyntaxException, AS4Exception;
```

Comment:
Her kaster den to, men i As4OptionalsBuilder.java kaster build kun en?

Reply:
Removed that exception 👍